### ISSUE TITLE
bug: White Screen Bug!!! important!!!

### ISSUE THREAD

**Issue by EyadIT94** [ionic-team/capacitor#3907]
### Capacitor Version

```
Installed Dependencies:

  @capacitor/ios not installed
  @capacitor/cli 2.4.4
  @capacitor/android 2.4.4
  @capacitor/core 2.4.4
  @capacitor/electron not installed

[success] Android looking great! 
```

### Platform(s)

Android

### Current Behavior

When I run the Android application using npx cap open android it's shows me a white screen when I run the application

2020-12-06 01:49:19.403 15837-15928/om.ayshay E/chromium: [ERROR:gl_surface_egl.cc(289)] eglChooseConfig failed with error EGL_SUCCESS
2020-12-06 01:49:19.557 15837-15894/om.ayshay E/chromium: [ERROR:service_manager.cc(156)] Connection InterfaceProviderSpec prevented service: content_renderer from binding interface: blink::mojom::BudgetService exposed by: content_browser
2020-12-06 01:49:20.932 15837-15837/om.ayshay E/Capacitor/Console: File: capacitor-runtime.js - Line 362 - Msg: ReferenceError: globalThis is not defined
2020-12-06 01:49:20.947 15837-15936/om.ayshay E/Capacitor: JavaScript Error: {"type":"js.error","error":{"message":"Uncaught ReferenceError: globalThis is not defined","url":"http://localhost/vendor-es2015.js","line":103386,"col":1,"errorObject":"{}"}}
2020-12-06 01:49:20.952 15837-15837/om.ayshay E/Capacitor/Console: File: http://localhost/vendor-es2015.js - Line 103386 - Msg: Uncaught ReferenceError: globalThis is not defined

### Expected Behavior

I should run the application and shows the pages

### Other Technical Details

`npm --version` output:
6.14.5
`node --version` output:
v12.16.3
`pod --version` output (iOS issues only):

### Additional Context

I search the same problem and I couldn't find a similar problem

**Comment by Ionitron** [comment]
This issue may need more information before it can be addressed. In particular, it will need a reliable Code Reproduction that demonstrates the issue.

Please see the Contributing Guide for [how to create a Code Reproduction](https://github.com/ionic-team/capacitor/blob/HEAD/CONTRIBUTING.md#creating-a-code-reproduction).

Thanks!
Ionitron 💙

**Comment by EyadIT94** [comment]
The error solved downgraded @angular/fire from 6.14 to 6.04 and the issue is solved, please close the bug.

Thanks 😊

**Comment by mobilityvalley** [comment]
I did it but it changes nothing. Do you have a clue ? 

I have these dependenices too:

    "firebase": "^8.1.1",
    "@capacitor-community/fcm": "^1.0.8",
